### PR TITLE
[WIP][Api/Tizen] using char rather than macro

### PR DESCRIPTION
**Pull request by helloahn**
Since the macro is not recognized, using gchar to pass the path

Signed-off-by: Hyoung Joo Ahn <hello.ahn@samsung.com>

**Self evaluation:**
1. Build test: [*]Passed [ ]Failed [ ]Skipped
2. Run test: [*]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: api/capi/src/nnstreamer-capi-tizen.c [hunk 1]
```diff
@@ -723,9 +724,9 @@ ml_tizen_mm_convert_element (ml_pipeline_h pipe, gchar ** result,
     }
 
     /* read ini, type CONFIGURE_TYPE_MAIN */
-    err =
-        _mmcamcorder_conf_get_info (0, (char *) MMFW_CONFIG_MAIN_FILE,
-        &cam_conf);
+    ini_name = g_strdup (MMFW_CONFIG_MAIN_FILE);
+    err = _mmcamcorder_conf_get_info (0, ini_name, &cam_conf);
+    g_free (ini_name);
```
Comment (myungjoo) [inline api/capi/src/nnstreamer-capi-tizen.c]: ```_mmcamcorder_conf_get_info``` accepts ```const char *```. How this fixes the issue?

Comment (myungjoo) [reply api/capi/src/nnstreamer-capi-tizen.c]: Ah.,... fix Line 94. You seem to have incorrect function signature there!
```
_mmcamcorder_conf_get_info (int type, char *ConfFile, camera_conf ** configure_info);
```
==>
```
_mmcamcorder_conf_get_info(MMHandleType handle, int type, const char *ConfFile, camera_conf **configure_info);
```

Comment (myungjoo) [reply api/capi/src/nnstreamer-capi-tizen.c]: Or we may include libmm's devel pkg and header if it's "Tizen".

**Comment by myungjoo** [comment]
First, test the fix by correcting the function declaration (extern ...)
If it is fixed by correcting function declaration while keep using macro, refactor the usage by BuildRequires: mmfw's devel pkg and mmfw's header if it's "Tizen" build.

**Comment by myungjoo** [comment]
If this header is not exported, let's do Step 1 only (keep using extern in our code)

And move to Step 2 later.

**Comment by jaeyun-jung** [comment]
@helloahn @myungjoo Sorry, I made a critical bug to parse the ini file. I added another PR (#1736) to fix this issue. Plz see the details and close this.

**Comment by helloahn** [comment]
Since critical issues are detected, it will be solved by the next prs.

**Pull request closed without merging**